.idea/codeStyles/Project.xml

```diff
@@ -161,4 +161,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Please don't push `.idea` file changes. You can simply delete this file from your PR.

Reply:
Done, I didn't know we could delete that, else I would've just done that instead of changing the code

Reply:
Wait, this is incorrect actually. What you've done might lead to deleting of the file from our codebase itself. I'll defer to @anandwana001 on this.

Reply:
I think the steps you should take here is:
1. Update your branch with the latest develop branch. (pull + push or fetch + merge + commit + push)
2. Now, the `.idea/codeStyles/Project.xml` will be added to your branch, as coming from the latest develop branch.
3. Now, you will see the file is same as this file https://github.com/oppia/oppia-android/blob/develop/.idea/codeStyles/Project.xml and no changes will be shown into your branch, meaning Android Studio is not updating this file. 
4. Now, in this PR you will only see 3 files changed which is our proto file where you had solved this issue.